app/Lib/enum.php

```diff
@@ -785,6 +785,18 @@ class TAndCLoginModeEnum
   const DisableAllServices = 'D';
 }
 
+class TapStatusEnum
```

Comment:
Sorry, there might have been some confusion, but we're not actually supporting the TAP statuses yet. (If/when we do, we'll need to add them to CoPerson and CoPersonRole.) Strictly speaking, the Core CO Person API is _not_ a TAP API, even though it looks very similar.

Reply:
Done